### PR TITLE
Stack_Using_STL

### LINKED ISSUE #2221: Stack using STL
### Description

Stack using STL

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by aman12207**
<hr>

## Description 📜

Added documentation on Stack in C++ STL.
Fixes #2737

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Stack_Using_STL.MD [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+The Standard Template Library (STL) in C++ contains the implementation of commonly known data structures such as arrays, lists, stacks, etc.
+To use the STL stack, you must first include the stack file:
+### `#include <stack>`
+
+<br>
+A Stack is a simple data structure for storing data.
+<br>
+
+In stack, the order in which the data arrives is important. The Stack allows elements to be added and removed from one end only. Stacks are ideal for enforcing sequential rules of access ie, LIFO.
```
Comment (deeqakkk) [inline C_CPP/Stack_Using_STL.MD:13]: do add "last in first out" after LIFO

**Review by deeqakkk — CHANGES_REQUESTED** [review]
also  update link to the directory folder

**Comment by aman12207** [comment]
> also update link to the directory folder

Please explain how can i do this

**Comment by aman12207** [comment]
@deeqakkk @Suvraneel  I have done the changes

**Comment by Suvraneel** [comment]
@aman12207  Wrong directory 
Correct directory :
C_CPP/Data Structures/Standard Template Library/Stack/

Shift doc there & update corresponding readme

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@aman12207 Wrong directory
Correct directory :
C_CPP/Data Structures/Standard Template Library/Stack/

Shift doc there & update corresponding readme

File: C_CPP/Data Structures/Stack/readme.md [hunk 2]
```diff
@@ -1,4 +0,0 @@
-- [Stacks](./Stacks.md)
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Stack/readme.md:1]: Restore this file

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by aman12207** [comment]
@Suvraneel Sir Please review this . I made changes as directed

**Comment by Suvraneel** [comment]
@aman12207  Good Work ❤️

**Pull request merged**